### PR TITLE
Adding documentation for TpmTool

### PR DESCRIPTION
**Pull request by ashleytqy**
Hi, the TPM team recently added a new in-box command line tool and would like to add a new page on the documentation site. Thanks!

Tagging @RonaldAi

### CODE REVIEW

File: WindowsServerDocs/administration/windows-commands/tpmtool.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# tpmtool
+
+This utility can be used to get information about the [Trusted Platform Module (TPM)](https://docs.microsoft.com/en-us/windows/security/information-protection/tpm/trusted-platform-module-overview).
+
+For examples of how to use this command, see [Examples](#BKMK_examples).
+
+## Syntax
+
+```
+tpmtool /Command [<Arguments>]
```
Comment (RonaldAi) [inline WindowsServerDocs/administration/windows-commands/tpmtool.md]: maybe use /parameter here, so the below section can be easily matched.

```suggestion
tpmtool /parameter [<Arguments>]
```

File: WindowsServerDocs/administration/windows-commands/tpmtool.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# tpmtool
+
+This utility can be used to get information about the [Trusted Platform Module (TPM)](https://docs.microsoft.com/en-us/windows/security/information-protection/tpm/trusted-platform-module-overview).
```
Comment (RonaldAi) [inline WindowsServerDocs/administration/windows-commands/tpmtool.md]: remove language specific code (en-us) from link
```suggestion
This utility can be used to get information about the [Trusted Platform Module (TPM)](https://docs.microsoft.com/windows/security/information-protection/tpm/trusted-platform-module-overview).
```

File: WindowsServerDocs/administration/windows-commands/tpmtool.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+```
+## Parameters
+
+|Parameter|Description|
+|---------|-----------|
+|getdeviceinformation|Displays the basic information of the TPM.|
+|gatherlogs [output directory path]|Collects TPM logs and places them in the specified directory. If path is not specified, will default to current directory.|
+|/?|Displays help at the command prompt.|
+
+## <a name="BKMK_examples"></a>Examples
```
Comment (RonaldAi) [inline WindowsServerDocs/administration/windows-commands/tpmtool.md]: why is the link called BKMK_examples? Is this copy-paste relic or generated automatically?

File: WindowsServerDocs/administration/windows-commands/tpmtool.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+To display the basic information of the TPM, type:
+```
+tpmtool getdeviceinformation
+```
+To collects TPM logs and place them in the current directory, type:
+```
+tpmtool gatherlogs
+```
+To collects TPM logs and place them in `C:\Users\Public\Desktop`, type:
```
Comment (RonaldAi) [inline WindowsServerDocs/administration/windows-commands/tpmtool.md]: c:\user\public\desktop does not work on my machine. Can you validate that this path actually works on your install and if not, find a path that does?

File: WindowsServerDocs/administration/windows-commands/tpmtool.md (hunk 3 above)
Comment (ashleytqy) [reply WindowsServerDocs/administration/windows-commands/tpmtool.md]: I copy-pasted. Will change to `tpmtool_examples`

**Comment by RonaldAi** [comment]
Hi @Justinha, you helped us with other TPM related documentation. Can you help us (Ashley and I) figure out who we should ping for approval to publish this new page? Thanks, Ron.

**Comment by Justinha** [comment]
@JasonGerend can you review and merge after Ashley adds metadata?

**Comment by ashleytqy** [comment]
#sign-off

**Comment by JasonGerend** [comment]
@eross-msft - this looks ready to publish - can you merge this, please?

**Pull request merged**